tests/test_http.py

```diff
@@ -1053,7 +1053,10 @@ def test_add_fail_for_resumable(self):
     upload = MediaFileUpload(
         datafile('small.png'), chunksize=500, resumable=True)
     self.request1.resumable = upload
-    self.assertRaises(BatchError, batch.add, self.request1, request_id='1')
+    with self.assertRaises(BatchError) as batch_error:
+      batch.add(self.request1, request_id='1')
+    self.assertEqual(str(batch_error.exception), '<BatchError "Media requests cannot be used in a batch request.">')
```

Comment:
Drop this line? I don't think we care that the string value of the exception raised by the system under test is some exact value.